jumanji/environments/routing/mmst/env.py

```diff
@@ -596,14 +597,11 @@ def num_connections(
             ratio_connections = total_connections / (n_comps - 1.0)
             return jnp.array([total_connections, ratio_connections])
 
-        connections = jnp.zeros((self.num_agents, 2))
-        for agent in range(self.num_agents):
-            total_ratio = num_connections(
-                state.nodes_to_connect[agent],
-                state.connected_nodes[agent],
-                self.num_nodes_per_agent,
-            )
-            connections = connections.at[agent].set(total_ratio)
+        connections = jax.vmap(num_connections, in_axes=(0, 0, None))(
```

Comment:
maybe outside the scope of this MR, but I would rename "num_connections" to something like get/calc _num_connections. Or even throw `_func` at the end of it

Reply:
That's a good point. I agree! For priority reasons, I will keep this way in this PR.